Improve ZHA pairing section (#33407)
home-assistant/home-assistant.github.io: source/_integrations/zha.markdown

### Best practices to avoid pairing/connection difficulties

> If you are having problems pairing a device to then ZHA integration then it is recommended to verify that you try to follow recommended best practices to avoid pairing and/or connection issues:

Reviewer: I think this can be cleaned up further, as there's a bit of unnecessary/duplicated information currently.

```suggestion
If you experience problems pairing a device, verify that you follow best practices to avoid pairing/connection issues:
```
